gcloud/storage/_helpers.py

```diff
@@ -82,7 +73,7 @@ def batch(self):
         """
         return _PropertyBatch(self)
 
-    def _reload_properties(self):
+    def reload(self):
```

Comment:
Artifact of JavaScript-y method chaining I would guess. I was just going for the rename here, but removing the return value is easy.

I checked locally and it causes no test failures.

Shall I ditch the return value?